qttime.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+extern "C" {
+#include <lua5.2/lua.h>
+#include <lua5.2/lualib.h>
+#include <lua5.2/lauxlib.h>
+}
+
+#include <QString>
+#include <QDateTime>
+#include <QtGlobal>
+
+int luaopen_qttime(lua_State *L);
```

Comment:
@LevchenkoS,
Please provide empty line in the end file source files.
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline